orchagent/muxorch.cpp

```diff
@@ -1208,6 +1244,47 @@ bool MuxOrch::handleMuxCfg(const Request& request)
     return true;
 }
 
+// Retrieve tc_to_queue_map and tc_to_dscp_map from CONFIG_DB, and 
+// resolve the ids from QosOrch
+bool MuxOrch::resolveQosTableIds()
+{
+    std::vector<FieldValueTuple> field_value_tuples;
+    if (cfgTunnelTable_.get(MUX_TUNNEL, field_value_tuples))
+    {
+        KeyOpFieldsValuesTuple tuple{"TUNNEL", MUX_TUNNEL, field_value_tuples};
+        for (auto it = kfvFieldsValues(tuple).begin(); it != kfvFieldsValues(tuple).end(); it++)
+        {
+            if (qos_to_ref_table_map.find(fvField(*it)) != qos_to_ref_table_map.end())
```

Comment:
We cannot use another orch's Data structure here. This must be handled in qosorch and retrieve from there.

Reply:
Agree